controller/api/public/grpc_server.go

```diff
@@ -211,9 +211,8 @@ func (s *grpcServer) TapByResource(req *pb.TapByResourceRequest, stream pb.Api_T
 	tapStream := stream.(tapServer)
 	tapClient, err := s.tapClient.TapByResource(tapStream.Context(), req)
 	if err != nil {
-		//TODO: why not return the error?
 		log.Errorf("Unexpected error tapping [%v]: %v", req, err)
-		return nil
+		return err
```

Comment:
hmmmmmmm is this branch not on latest master?
this change was already made in https://github.com/linkerd/linkerd2/pull/1371/files